.gitignore

```diff
@@ -0,0 +1,2 @@
+*.pyc
+log
```

Comment:
ignorance of `*.pyc` is OK, but the `log` directory **may** be used to reserve  source code for logging.
if necessary, u'd better configure it in `.git/info/exclude` for local ignorance.

Reply:
Done.